src/main/java/com/notably/logic/suggestion/SuggestionEngine.java

```diff
@@ -0,0 +1,14 @@
+package com.notably.logic.suggestion;
+
+/**
+ * Represents the Suggestion Engine instance.
+ * This is the class that we interact with when the user is typing an input.
```

Comment:
This second line "This is ..." is not really descriptive, don't you feel? How about rewriting it as "This is the class that manages the generation of suggestions"?